### ISSUE TITLE
Unqualified exec path throws error

### ISSUE THREAD

**Issue by bplunkert** [inkblot/puppet-terraform#5]
>        Error: Validation of Exec[/usr/local/bin/terraform-version] failed: 'mv /tmp/terraform /usr/local/bin/terraform-0.7.13' is not qualified and no path was specified. Please qualify the command or specify a path. at /tmp/kitchen/modules/terraform/manifests/init.pp:37

I believe we can correct this by adding the path attribute to the exec resource.

**Comment by mattford63** [comment]
Yeah just hit this also in later versions of puppet.  Adding path fixes.

``` 
exec { '/usr/local/bin/terraform-version':
  path        => [ '/bin' ],
   command     => "mv ${tmp_dir}/${bin_name} ${target_dir}/${bin_name}-${version}",
    subscribe   => Archive["${tmp_dir}/${archive_filename}"],
    refreshonly => true,
  }
```

**Comment by bplunkert** [comment]
This should be resolved by #4
